lib/matplotlib/ticker.py

```diff
@@ -444,20 +449,31 @@ class ScalarFormatter(Formatter):
 
     """
 
-    def __init__(self, useOffset=None, useMathText=None, useLocale=None):
+    def __init__(self, useOffset=None, useMathText=None, useLocale=None, usetex=None):
```

Comment:
```suggestion
    def __init__(self, useOffset=None, useMathText=None, useLocale=None, *, usetex=None):
```

If we are adding this, might as well add it key-word only to start.

Reply:
> If we are adding this, might as well add it key-word only to start.

TBH I'm not sure I understand this subtlety, but I did this change anyway.